ZbW_P_Contact_Manager/Models/Person.cs

```diff
@@ -1,14 +1,12 @@
-﻿using Microsoft.VisualBasic.ApplicationServices;
-using System.Collections;
+﻿using System.Collections;
 using System.ComponentModel;
-using System.Data;
 using System.Reflection;
 
 namespace Model
 {
     public class Person
     {
-        public Guid Id { get; set; }
+        public Guid Id { get; set; } = Guid.NewGuid();
```

Comment:
I'm not sure what this does, but wouldn't this create a new GUID on each instance creation? This would probably break alot of code, as we loose the instances when writing and reading from csv